tests/integration/api_v2/http/votesReceived.test.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable quote-props */
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { api } = require('../../../helpers/api');
+
+const baseUrl = config.SERVICE_ENDPOINT;
+const baseUrlV2 = `${baseUrl}/api/v2`;
+
+const {
+	badRequestSchema,
+	notFoundSchema,
+	wrongInputParamSchema,
+	metaSchema,
+} = require('../../../schemas/httpGenerics.schema');
+
+const {
+	voterSchemaVersion5,
+} = require('../../../schemas/voter.schema');
+
+[
+	`${baseUrlV2}/votes_received`,
+	// `${baseUrlV1}/voters`, TODO : Added support for api/v2/votes
```

Comment:
I think we do not plan to add support for this endpoint. Should be safe to remove. @MichalTuleja please confirm.
```suggestion
```